packages/strapi-admin/validation/common-validators.js

```diff
@@ -94,9 +102,9 @@ const updatePermissions = yup
             checkNoDuplicatedPermissions
           )
           .test(
-            'delete-fields-are-null',
-            'The action "plugins::content-manager.explorer.delete" must have fields set to null or undefined',
-            checkCTPermsDeleteHaveFieldsToNull
+            'fields-restriction',
+            'The actions must have fields set to null or undefined',
+            checkContentTypesFieldsRestriction
```

Comment:
I think you should use the this.createError() of the test function to set a better message with the action name in it.